### PR TITLE
Modified some tag colors

### PR DESCRIPTION
**Pull request by simonegramegna**

### CODE REVIEW

**Comment by gabrieldavid98** [comment]
Hey!

First Thanks for contributing

i'm gonna check the new colors and then i'm going to merge pull request

Greetings ✋

File: theme/theme.less [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@bgColor: #000;
-@fgColor: #01DF3A;
+@bgColor: black;
+@fgColor: #71F20;
```
Comment (gabrieldavid98) [inline theme/theme.less]: Hey

It looks like hex num isn't complete, please fix it and give me the correct color :D

File: theme/theme.less [hunk 2]
```diff
@@ -44,10 +44,10 @@
     }
     
     .cm-string, .cm-string-2, .cm-hr {
-        color: #01DF01;
+        color: #20FF00;
     }
     
-    .cm-variable-2{
+    .cm-variable-2
```
Comment (gabrieldavid98) [inline theme/theme.less]: Hey

Her open brace is required

File: theme/theme.less [hunk 3]
```diff
@@ -47,7 +47,7 @@
         color: #20FF00;
     }
     
-    .cm-variable-2
+    .cm-variable-
```
Comment (gabrieldavid98) [inline theme/theme.less]: Hey

Here brace is required, and it seems that "2" is missing in class name please fix this

**Review by gabrieldavid98 — COMMENTED** [review]
Hello

I was checking the pull request and i found some errors, please fix it and then re send pull request.

Greetings ✋

**Comment by simonegramegna** [comment]
Ok...i'm gonna to fix those errors

**Pull request closed without merging**